srtp/srtp.c

```diff
@@ -82,13 +82,14 @@ srtp_debug_module_t mod_srtp = {
 static srtp_err_status_t srtp_validate_rtp_header(void *rtp_hdr,
                                                   int *pkt_octet_len)
 {
-    if (*pkt_octet_len < octets_in_rtp_header)
-        return srtp_err_status_bad_param;
-
     srtp_hdr_t *hdr = (srtp_hdr_t *)rtp_hdr;
 
     /* Check RTP header length */
     int rtp_header_len = octets_in_rtp_header + 4 * hdr->cc;
+
+    if (*pkt_octet_len < octets_in_rtp_header)
```

Comment:
If the packet length is too short, the line above that reads into the header can cause an access violation.  We need to keep this line at the beginning of the function.